content/en/docs/components/training/mxnet.md

```diff
@@ -46,11 +42,11 @@ Check that the Apache MXNet operator is running via:
 kubectl get pods
```

Comment:
Since we are installing manager under kubeflow namespace.
```suggestion
kubectl get pods -n kubeflow
```

Reply:
Nice catch.